src/server/bg_services/usage_aggregator.js

```diff
@@ -112,6 +112,9 @@ async function background_worker() {
     const till = moment().subtract(8, 'weeks').valueOf();
     dbg.log0('Deleting reports older than', new Date(till));
     await EndpointStatsStore.instance.clean_bandwidth_reports({ till });
+
+    // Clean endpoint group reports here as well?
+    await EndpointStatsStore.instance.clean_endpoint_group_reports({ till });
```

Comment:
Can you also update the function inside this function from _format_endpoint_gorup_report_query to _format_endpoint_group_report_query
https://github.com/noobaa/noobaa-core/blob/4edc2a413b5573b3093ea7aa62826896ea23930f/src/server/analytic_services/endpoint_stats_store.js#L180-L181
it's just a very annoying typo...